src/channel/index.ts

```diff
@@ -908,7 +909,7 @@ export default class Channel {
       gasPrice: number | BigNumber;
       gasUsed: number | BigNumber;
       height: number;
-      log: string;
+      log: string | [Event];
```

Comment:
`[Event]` as I know is a single-size tuple with Event, but there can be multiple events as in your example
also, I think `string` was added there by a mistake, so I'm replacing it with `Event[]`

Reply:
Also, `callerId`, `callerNonce`, `contractId` is missed here, but they a defined in a `ContractCallObject`, so I would use it instead. The only problem is that `ContractCallObject` uses `string` instead of `xx_`-prefixed strings, it has to be fixed in Node Api generator 🙁